data/json/requirements/ammo.json

```
@@ -63,5 +63,121 @@
     "type": "requirement",
     "//": "Components required for .270 Winchester ammo",
     "components": [ [ [ "270win_casing", 1 ] ], [ [ "lgrifle_primer", 1 ] ] ]
+  },
+  {
+    "id": "22_casehead",
+    "type": "requirement",
+    "//": "References needed for boltfaces and magazines using .22 lr or derivatives",
+    "tools": [
+      [
+        [ "22_casing", -1 ],
+        [ "22_casing_new", -1 ],
+        [ "22_fmj", -1 ],
+        [ "22_lr", -1 ],
+        [ "22_cb", -1 ],
+        [ "22_ratshot", -1 ],
+        [ "reloaded_22_lr", -1 ],
+        [ "reloaded_22_fmj", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "223_casehead",
+    "type": "requirement",
+    "//": "References needed for boltfaces and magazines using .223 or derivatives",
+    "tools": [
+      [
+        [ "300blk_casing", -1 ],
+        [ "223_casing", -1 ],
+        [ "223", -1 ],
+        [ "556", -1 ],
+        [ "556_incendiary", -1 ],
+        [ "reloaded_223", -1 ],
+        [ "reloaded_556", -1 ],
+        [ "reloaded_556_incendiary", -1 ],
+        [ "300blk", -1 ],
+        [ "300blk_ss", -1 ],
+        [ "reloaded_300blk", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "9mm_casehead",
+    "type": "requirement",
+    "//": "References needed for boltfaces and magazines using .30-06 or derivatives",
```

Seems like copy-paste error.
```suggestion
    "//": "References needed for boltfaces and magazines using 9mm or derivatives",
```

---

caught red-handed.